python/src/MethodOfMomentsFactory_doc.i.in

```diff
@@ -14,22 +14,49 @@ Notes
 -----
 This method fits a scalar distribution to data of dimension 1, using the method of moments.
 
-Let :math:`(\vect{x}_1, \dots, \vect{x}_n)` denote the sample, :math:`F_{\vect{\theta}}`
+Let :math:`(\vect{x}_k)_{1 \leq k \leq n}` be
+the sample, :math:`F_{\vect{\theta}}`
 the  cumulative distribution function we want to fit to the sample,
-and :math:`\vect{\theta} \in  \Theta \subset\Rset^p`  its parameter vector.
+and :math:`\vect{\theta} \in  \Theta \subset\Rset^K`  its parameter vector.
```

Comment:
It seems to me that a roman upcase capital letter should be used primarily for random variables.
Since $d$ is now the standard for "dimension", I suggest this.
```suggestion
and :math:`\vect{\theta} \in  \Theta \subset\Rset^{d_\theta}`  its parameter vector
where :math:`{d_\theta} \in Nset` is the number of parameters.
```